Packages/com.unity.inputsystem/InputSystem/Plugins/Android/AndroidGameController.cs

```diff
@@ -152,13 +168,13 @@ namespace UnityEngine.InputSystem.Android
     /// Other exotic gamepads have different mappings
     ///  Xbox Gamepad (for ex., Microsoft X-Box One pad (Firmware 2015)) mapping (Note mapping: L2/R2/Right Stick)
     ///  Left Stick -> AXIS_X(0) / AXIS_Y(1)
-    ///  Right Stick -> AXIS_RX (12) / AXIS_RY(13)
+    ///  Right Stick -> AXIS_Z (11) / AXIS_RZ(14)
```

Comment:
So it seems, this comment is no longer relevent, as mapping matches the comment scope above next to /// Most of the gamepads: 

if so, can you delete this scope, and add Xbox Controller in the list above (after Sony Interactive Entertainment Wireless), also could you please add device name and android os version next to the device just as an additional info